### DISCUSSION TITLE
No devices after installing

### DISCUSSION THREAD

**Discussion by baenaeduardo** [mattieserver/netbox-topology-views#206] [Q&A]
Hi , 

Just installed this plugin version 0.5.3 as we are on Netbox Version (v2.9.4). 

The installation seems to have gone well following the instructions on the readme page, but I can't see any devices in the topology view. 

After reading every issue on this plugin and on other plugins and trying different solutions , I've given up and decided to post this question. 

Thanks

The console on chrome shows this error: 

![image](https://user-images.githubusercontent.com/5090199/176714478-9ebff600-cdd2-4886-8b5d-c37da16ec2d0.png)

![image](https://user-images.githubusercontent.com/5090199/176714217-647f8986-5488-4e08-b5c3-99064681c903.png)

**Comment by mattieserver** [comment]
Can you try version '[v0.4.10](https://github.com/mattieserver/netbox-topology-views/releases/tag/v0.4.10)' ?

**Comment by baenaeduardo** [comment]
Hi, 

Unfortunately changing versions didnt work. 

(venv) root@aim-netbox:/opt/netbox# pip list |grep 'netb'
netbox-topology-views (0.4.10)
(venv) root@aim-netbox:/opt/netbox# 

This is one of the messages I'm still getting on chrome. 
http://10.254.201.144/api/plugins/topology-views/preselecttags/ 404 (Not Found)

Thanks

**Comment by dreng** [comment]
Converted this into a discussion as it seems to be an individual problem that is not related to the current version.
